docs/pages/concepts/logical/relationships.md

```diff
@@ -20,15 +20,16 @@ Meshery supports a variety of relationships between components. Relationships ar
 
 Here is a list of the different types of relationships that Meshery supports:
 
-1. Edge
-   1. Network
-   1. Firewall
-   1. Binding
-      1. Mount
-      1. Permission
-1. Heirarchical
-   1. Inventory
-   1. Parent
+1. `Edge` : Edge relationships indicate the possibility of traffic flow between two components. Edge relationship can be divided into 3 sub types. 
```

Comment:
Look for the description for these in contributing docs and expand upon them.
The current descriptions for Network, Firewall... is not enough.